File: Binary_to_Decimal_Biway_Converter.cs
```CSharp
using System;

class Program
{
    static void Main(string[] args)
    {
        while (true)
        {
            Console.WriteLine("1. Binary to Decimal");
            Console.WriteLine("2. Decimal to Binary");
            Console.WriteLine("3. Exit");
            Console.Write("Enter your choice: ");
            int choice = Convert.ToInt32(Console.ReadLine());

            switch (choice)
            {
                case 1:
                    Console.Write("Enter a binary number: ");
                    string binary = Console.ReadLine();
                    Console.WriteLine("Decimal: " + Convert.ToInt32(binary, 2));
                    break;

                case 2:
                    Console.Write("Enter a decimal number: ");
                    int decimalNumber = Convert.ToInt32(Console.ReadLine());
                    Console.WriteLine("Binary: " + Convert.ToString(decimalNumber, 2));
                    break;

                case 3:
                    Environment.Exit(0);
                    break;

                default:
                    Console.WriteLine("Invalid choice");
                    break;
            }
        }
    }
}
```

File: Binary_to_Decimal_Biway_Converter.py
def binary_to_decimal(binary):
    return int(binary, 2)

def decimal_to_binary(decimal):
    return bin(decimal).replace("0b", "")

while True:
    print("1. Binary to Decimal")
    print("2. Decimal to Binary")
    print("3. Exit")
    choice = int(input("Enter your choice: "))
    if choice == 1:
        binary = input("Enter a binary number: ")
        print("Decimal: ", binary_to_decimal(binary))
    elif choice == 2:
        decimal = int(input("Enter a decimal number: "))
        print("Binary: ", decimal_to_binary(decimal))
    elif choice == 3:
        break
    else:
        print("Invalid choice")